### PR TITLE
[WIP] wxWidgets3.1:wxWidgets3.2: Upgrade to 3.2.0 and

### PR DESCRIPTION
**Pull request by stahta01**
convert to split packages wxmsw, wxconfig, bakefile, locale, and wxrc.
Also, prevent CLANG32 from using "-mthreads". The split package
msw_cb_headers added to aid in building the IDE Code::Blocks. Also, add
wxWidgets3.2 meta package that depends on wxmsw, wxrc, locale, and
wxconfig. And, rename folder to mingw-w64-wxWidgets3.2 from
mingw-w64-wxWidgets3.1. Changed to CXX standard 17.

### CODE REVIEW

File: mingw-w64-wxWidgets3.2/PKGBUILD [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
+_libname=wxmsw
+_wx_basever=3.2
 # Example _wx_buildver value is "-rc"
 _wx_buildver=
 
-pkgbase=mingw-w64-${_basename}${_wx_basever}
-pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}${_wx_basever}"
-pkgver=${_wx_basever}.7
-pkgrel=2
+pkgbase=mingw-w64-${_realname}3.2
```
Comment (mmuetzel) [inline mingw-w64-wxWidgets3.2/PKGBUILD:17]: End with `${_wx_basever}` here, too?

File: mingw-w64-wxWidgets3.2/usage_notes.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Suggested usage notes:
+
+Add "${MINGW_PACKAGE_PREFIX}-wxmsw3.2" to depends.
+And in some/most cases, add "${MINGW_PACKAGE_PREFIX}-wxWidgets3.2"
+to makedepends. In very rare cases, it might be necessary to add 
+"${MINGW_PACKAGE_PREFIX}-bakefile3.2" to makedepends. When building
+Code::Blocks, the adding of "${MINGW_PACKAGE_PREFIX}-msw_cb_headers3.2"
```
Comment (mmuetzel) [inline mingw-w64-wxWidgets3.2/usage_notes.txt:7]: I'm not a native speaker. But that sounds a bit of to me.
Replace `the adding of` with `adding`?

Comment (mmuetzel) [reply mingw-w64-wxWidgets3.2/usage_notes.txt:7]: You use the imperative voice in line 4 of this file. That sounds "more English" imho. You could use it here, too.

Btw, I like the fact that you documented how the split up parts should be used correctly. 👍

Comment (stahta01) [reply mingw-w64-wxWidgets3.2/usage_notes.txt]: Rewrote the last sentence; I hope it sounds right.

File: mingw-w64-wxWidgets3.2/PKGBUILD (hunk 1 above)
Comment (stahta01) [reply mingw-w64-wxWidgets3.2/PKGBUILD:17]: I did not do that to help me remember to change it when the folder is renamed to just "mingw-w64-${_realname}".

Comment (mmuetzel) [reply mingw-w64-wxWidgets3.2/PKGBUILD:17]: Sounds reasonable.

**Comment by stahta01** [comment]
Decided this package release would be a good time to add wxgtk library closing till I get it done.

**Pull request closed without merging**